evennia/utils/opclasses.py

```diff
@@ -0,0 +1,319 @@
+import datetime as _dt
+from evennia.utils.ansi import ANSIString as _ANSI
+from evennia.utils.validfuncs import _TZ_DICT
+from evennia.utils.valid import VALID_HANDLER as _VAL
+
+
+class _BaseOption(object):
+    """
+    Abstract Class to deal with encapsulating individual Options. An Option has a name/key, a description
+    to display in relevant commands and menus, and a default value. It saves to the owner's Attributes using
+    its Handler's save category.
+
+    Designed to be extremely overloadable as some options can be cantankerous.
+
+    Properties:
+        expect_type (str): What users will see this as asking for. Example: Color or email.
+        valid: Shortcut to the loaded VALID_HANDLER.
+        valid_type (str): The key of the Validator this uses.
+    """
+    expect_type = ''
+    valid = _VAL
+    valid_type = ''
```

Comment:
It would be clearer if this was named `validator_key` or similar since the point of it is to look up and call a validator by this name.